src/pages/task/TaskTabs.tsx

```diff
@@ -96,6 +99,12 @@ export const TaskTabs: React.FC<TaskTabProps> = ({ task, taskFiles }) => {
         key="execution-tasks-tab"
       >
         <TasksTable
+          tableChangeHandler={(...[, , sorter]) => {
+            updateQueryParams({
+              sorts: toSortString(sorter),
+              [RequiredQueryParams.Execution]: "0",
```

Comment:
We can't hardcode execution since tasks can have multiple executions this will cause it to always show the first one.